geopandas/tests/test_geodataframe.py

```diff
@@ -393,6 +393,24 @@ def test_to_file_schema(self):
 
         assert result_schema == schema
 
+    def test_to_file_crs(self):
+        """
+        Ensure that the file is written according to the crs
+        if it is specified
+
+        """
+        tempfilename = os.path.join(self.tempdir, 'crs.shp')
+        self.df.to_file(tempfilename, crs=self.df.crs)
+        # Read layer back in
+        df = GeoDataFrame.from_file(tempfilename)
+        assert 'geometry' in df
+        assert len(df) == 5
+        assert np.alltrue(df['BoroName'].values == self.boros)
```

Comment:
I'd probably just remove the three lines above, which aren't all that relevant to the feature being tested here.

Reply:
ok